utils/config.py

```diff
@@ -61,6 +61,34 @@ def get_deployer_account() -> Union[LocalAccount, Account]:
     return accounts.load(os.environ["DEPLOYER"]) if (is_live or "DEPLOYER" in os.environ) else accounts[4]
 
 
+def get_web3_storage_token() -> str:
+    is_live = get_is_live()
+    if is_live and "WEB3_STORAGE_TOKEN" not in os.environ:
+        raise EnvironmentError("Please set WEB3_STORAGE_TOKEN env variable to the web3.storage api token")
+
+    return os.environ["WEB3_STORAGE_TOKEN"] if (is_live or "WEB3_STORAGE_TOKEN" in os.environ) else ""
+
+
+def get_infura_io_keys() -> Tuple[str, str]:
```

Comment:
Most likely I will delete the alternative, to not confuse people. This was added to check that  we could generate the same `cid` by different services. now requests from UI go through web3.storage as well, so it will be smoother if we will use it.

Reply:
remove alternative